### ISSUE TITLE
Logging support?

### ISSUE THREAD

**Issue by gpakosz** [DTVD/rainbowstream#158]
Hello,

Do you plan to add logging to Rainbowstream?

Thanks.

**Comment by the4ndy** [comment]
what are you looking to have logged? Can you elaborate on what you would like to see out of logging.  
  
To be fair, I am in NO WAY the owner or curator or even a contributor of this project. I am just curious and would also enjoy some logging (ideally with syslog support). I would want it to log, essentially everything it did, sometimes I want to know what happened in the background when i do a certain thing. I am having a hard time making a solid argument for a NEED for logging, but its definitely a desire of mine as well.

**Comment by DTVD** [comment]
what kind of logging do you want ? I'm curious and don't really see a need at any situation.

**Comment by gpakosz** [comment]
Same kind of logs are IRC logs so that I can keep the rainbowstream idle and get to read the log later.

**Comment by gpakosz** [comment]
Hello folks, getting back to this question after 2 years.

What I really need is a way to let tweets accumulate until I read them all or eventually decide to let them go.

So that e.g. at the end of my work day, I know there are ~200 unread tweets. I can decide to read them all one by one, skim through them rapidly or skip them all (~ mark them all as read). Do I make sense?

No twitter client seems to behave this way or maybe can rainbowstream fill that need with pausing the stream?

**Comment by carc1n0gen** [comment]
Someone correct me if I'm wrong, but I'm not sure this is the right tool for what you want.  In my experience, rainbowstream always stops getting new tweets after a certain idle time

**Comment by gpakosz** [comment]
@carc1n0gen good point, I'm just getting back to rainbowstream out of frustration from other clients. I will need to check that out
